js/inject-lai-webapp.js

```diff
@@ -0,0 +1,31 @@
+function getWebappJSPathFromHtml(webappPage) {
+  // For production environment
+  var webappJSPath = (/"(\/static\/js\/main\.[\da-z]+\.js)"/.exec(webappPage) || [])[1];
+  if (webappJSPath) {
+    return webappJSPath;
+  }
+  // For local environment
+  return (/"(\/static\/js\/bundle\.js)"/.exec(webappPage) || [])[1];
+}
+
+(function() {
+  // Using the user's homepage because this way it won't break
+  // if we replace the homepage with webflow or WP
+  fetch("/injectwebapp")
```

Comment:
Since this will be used on a domain other than lightning.ai, we should hardcode it here.

```suggestion
  fetch("https://lightning.ai/injectwebapp")
```